AdamBignell/RelNet
# Testing, model state, and generating all data

## To test: 
```
python3 relation.py --test-all            #NLL
python3 relation.py --test-all --BCE      #BCE
```
Make sure to change your DATA_DIR, BEST_BCE, and BEST_NLL constants
#### But first you must generate all data
```
python3 generate_all_data.py
```
Make sure to change your DATA_DIR
